massa-protocol-worker/src/protocol_worker.rs

```diff
@@ -144,6 +146,11 @@ pub struct ProtocolWorker {
     checked_operations: Set<OperationId>,
     /// List of processed headers
     checked_headers: Map<BlockId, BlockInfo>,
+    /// List of operations received
+    // TODO: Operation should be store when receive and we work only with id in the future.
```

Comment:
TODO: OperationS should be storeD when receiveD and we work only with id in the future.

Reply:
Done.